src/Components/Modals/EditAdditionalRepositoriesModal.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Modal, ModalVariant, Button } from '@patternfly/react-core';
+import EditAdditionalRepositoriesTable from '../EditAdditionalRepositoriesTable/EditAdditionalRepositoriesTable';
+
+const EditAdditionalRepositoriesModal = (props) => {
+  const { handleModalToggle, isOpen, repositories } = props;
+
+  const editAdditionalRepositoriesDescription =
+    'The core repositories for your operating system version, for example BaseOS and AppStream, are always enabled and do not need to be explicitly added to the activation key.';
+
+  /* Modal Return with attributes */
+
+  return (
+    <React.Fragment>
+      <Modal
+        variant={ModalVariant.large}
+        title="Edit additional repositories"
+        description={editAdditionalRepositoriesDescription}
+        isOpen={isOpen}
+        onClose={handleModalToggle}
+        actions={[
+          <Button
+            key="Save changes"
+            variant="primary"
+            onClick={handleModalToggle}
+          >
+            Save Changes
+          </Button>,
+          <Button key="cancel" variant="link" onClick={handleModalToggle}>
+            Cancel
+          </Button>,
+        ]}
+      >
+        <EditAdditionalRepositoriesTable repositories={repositories} />
```

Comment:
you're using isLoading and error props in EditAdditionalRepositoriesTable, but never pass them